packages/kbn-ui-framework/dist/kui_light.css

```diff
@@ -57,13 +57,8 @@
  * 5. Fixing this bug means we now need to align the children to the right.
  */
 :focus:not([class^="eui"]) {
-  z-index: 1;
-  /* 1 */
-  outline: none !important;
-  /* 2 */
-  -webkit-box-shadow: 0 0 0 1px #FFF, 0 0 0 2px #006BB4;
-          box-shadow: 0 0 0 1px #FFF, 0 0 0 2px #006BB4;
-  /* 3 */ }
+  -webkit-animation: 350ms cubic-bezier(0.694, 0.0482, 0.335, 1) 1 normal forwards focusRingAnimate;
+          animation: 350ms cubic-bezier(0.694, 0.0482, 0.335, 1) 1 normal forwards focusRingAnimate; }
```

Comment:
Any worry that this will be negated by the advanced settings no animation option?

Reply:
I tested. Still works fine. Animations just end on the last part of the animation from what I can tell. 

![image](https://user-images.githubusercontent.com/324519/54053413-01a7f980-419c-11e9-956b-6fbbeb50ab4d.png)